Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,125 @@
+
+Table 2: Power Requirements.<br />
+
+For this system, five volts is required for all components, so the five volt power supplies will be able to provide power to all the components. The worst case power draw from the system is 45.67 W, so the chosen 100 W power supplies will meet those for the motors with plenty of wattage to spare. The cost reduction from a lower wattage power supply is minimal, so we will have some extra power to account for any additional functionality or errors in the camera system. This power supply also gives two outputs at 5 V, which was not common at this price point.  The Raspberry Pi 5 V 15 W power supply will provide power to the Raspberry Pi 4B. This is the power supply designed by Raspbery Pi for the Raspberry Pi 4B, so it perfectly matches the power requirements. <br /> 
+
+##  DC 5 V 20 A 100 W Power Supply
+The maximum power requirements for the motors is 29.92 W, so the 100 W power supply will easily cover that. At first glance, it may seem like overkill, but this power supply is very useful for meeting this subsystems's requirements. Power supplies with multiple 5 V output terminals are not common in the market, so this option allowed the design team to use a single power supply to provide power to both motors. Also, the price point on this power supply was very similar, if not cheaper, than other power supplies with only one 5 V output. It also includes safety features such as shortage, overload, and over voltage protections, and is safety compliant with CE, RoHS and FCC. An issue with design choice to use power supplies, is it limits the design team's possible locations to house the Camera system. However, having more location flexibility with an alternate design such as a solar charged battery would be very expensivewith the maximum power for the system nearing 50 W. The design team decided that the cost-benefit analysis would not be worth it to implement a solar solution for this subsystem.
+<br />
+##  Gratury Junction Box, IP67 Waterproof Plastic Enclosure
+The Gratury Junction Box, IP67 Waterproof Plastic Enclosure will provide protection to the system from the elements. This NEMA box is rated at IP67 which protects the system from rain, snow, dust, wind, and UV. The design team chose this option to ensure the safety of the many sensitive electrical components from a variety of weather conditions. This NEMA box is also similarly priced to competiors, but this product was the closest sized options for our need at the lowest price.
+<br />
+##   16 AWG Wire
+16 AWG wire is required to comply with ASTM B258-18, The standard for wire gauge size based on current through a conductor.  Since the maximum amperage for 16 gauge wire is 3.7 A and the maximum amperage for our system is 3 A, the design is in compliance with the standard. With the amount of wire purchased and the high rating, this wire can also for other subsystems in the project if needed, such as the receiver power system. <br />
```

Comment:
Same comment as before about the standard.

Reply:
Standard changed